### PR TITLE
show shortcuts in advanced digitizing tools tooltips + fix small typo

### PR DESCRIPTION
**Pull request by tudorbarascu**
I couldn't find any docs regarding the advanced digitizing panel shortcuts and I lost quite a bit of time finding them out.
They should now be fully presented in the tooltips.

untitled

Any suggestions are welcomed.

### CODE REVIEW

File: src/gui/qgsadvanceddigitizingdockwidget.cpp [hunk 1]
```diff
@@ -56,7 +56,7 @@ bool QgsAdvancedDigitizingDockWidget::lineCircleIntersection( const QgsPoint& ce
 
   if ( disc < 0 )
   {
-    //no intersection or tangeant
+    //no intersection or tangent
```
Comment (m-kuhn) [inline src/gui/qgsadvanceddigitizingdockwidget.cpp]: Can you add that to `scripts/spell_check/spelling.dat`?

**Comment by m-kuhn** [comment]
Good job!

**Comment by NathanW2** [comment]
I have recently added a commit showing shortcuts in all toolbar buttons.
Are you follow that visual tooltip style convention here as well.

If you hover over a toolbar button you can see an example.  Just makes for
better UX if things are the same everywhere.

File: src/gui/qgsadvanceddigitizingdockwidget.cpp (hunk 1 above)
Comment (tudorbarascu) [inline src/gui/qgsadvanceddigitizingdockwidget.cpp]: sure! thank!

**Comment by tudorbarascu** [comment]
@NathanW2 Thanks for the tip!

It should be good now.

untitled

Do you think I should have just presented the shortcuts between ``()``?
With beginners in mind, I'm leaning towards presenting the shortcuts explicitly, e.g. ``press c for..`` instead of a simple ``c``.

**Comment by NathanW2** [comment]
@tudorbarascu that is fine now.  Was just about having the shortcut part inside `()` and bold text so that looks good to me.

**Comment by tudorbarascu** [comment]
@m-kuhn @NathanW2 Gentle ping for merging..

**Comment by m-kuhn** [comment]
@NathanW2 your call

**Comment by NathanW2** [comment]
Thanks. Yes looks great. I forgot the merge the other day

**Pull request merged**